.azure-pipelines/pull_request.yml

```diff
@@ -148,6 +148,7 @@ stages:
                 -DPYTHON_EXECUTABLE=$(command -v python) \
                 -DCMAKE_INSTALL_PREFIX="$(install_dir)" \
                 -DCMAKE_BUILD_TYPE=Release \
+                -DCMAKE_INTERPROCEDURAL_OPTIMIZATION=OFF \
```

Comment:
This change isn't mirrored in `conda/build.sh`?

Reply:
Yes, wondering whether the build inside the conda env will work? Maybe we can just try when merging into master?

Reply:
I suspect they won't but as we know what the fix is, it should be easy to implement if needed.